packages/vats/src/vat-bank.js

```diff
@@ -316,7 +316,18 @@ export function buildRootObject(_vatPowers) {
             `Only fungible assets are allowed, not ${assetKind}`,
           );
 
-          const assetRecord = harden({ ...kit, denom, brand });
+          // Create an escrow purse for this asset, seeded with the payment.
+          const escrowPurse = E(kit.issuer).makeEmptyPurse();
```

Comment:
I don't see tests for this. Is it expensive to test? Is it covered by existing tests?

Reply:
Good point.  I'll make some tests.